documentation/Installing-Clear-Containers-on-RHEL.md

```diff
@@ -1,206 +1,36 @@
-This is required for `gcc`.
-
-```
-$ curl -L -O ftp://gcc.gnu.org/pub/gcc/infrastructure/mpc-1.0.3.tar.gz
-$ tar -xvf mpc-1.0.3.tar.gz
-$ pushd mpc-1.0.3
-$ ./configure
-$ make && sudo make install
-$ popd
-
-$ curl -L -O ftp://gcc.gnu.org/pub/gcc/infrastructure/gmp-6.1.0.tar.bz2
-$ tar -xvf gmp-6.1.0.tar.bz2
-$ pushd gmp-6.1.0
-$ ./configue
-$ make && sudo make install
-$ popd
+$ ./installation/rhel-setup.sh
 
-$ curl -L -O ftp://gcc.gnu.org/pub/gcc/infrastructure/mpfr-3.1.4.tar.bz2
-$ tar -xvf mpfr-3.1.4.tar.bz2
-$ pushd mpfr-3.1.4
-$ ./configure
-$ make && sudo make install
-$ popd
-
-```
-
-## Install updated gcc
-
-This is required for `qemu-lite`.
-
-```
-$ sudo yum install gmp-devel mpfr-devel libmpc-devel gcc-c++
-$ curl -L -O http://www.mirrorservice.org/sites/ftp.gnu.org/gnu/gcc/gcc-6.2.0/gcc-6.2.0.tar.gz
-$ tar xvf gcc-6.2.0.tar.gz
-$ pushd gcc-6.2.0
-$ ./configure --enable-languages=c --disable-multilib --prefix=/usr/local/gcc-6.2.0
-$ make -j5 && sudo make install
-$ popd
-
-```
-
-## Install qemu-lite
-
-Although this package installs the `qemu-lite` version of `qemu`, the binary will still be called `qemu-system-x86_64`.
-Requires at least `gcc` 5.
-
-```
-$ export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH
-$ export PATH=/usr/local/gcc-6.2.0/bin:$PATH
-$ sudo yum install libcap-ng-devel pixman-devel libattr-devel libcap-devel
-$ curl -L -O https://github.com/01org/qemu-lite/archive/741f430a960b5b67745670e8270db91aeb083c5f.tar.gz
-$ tar xvf 741f430a960b5b67745670e8270db91aeb083c5f.tar.gz
-$ pushd qemu-lite-741f430a960b5b67745670e8270db91aeb083c5f
-$ CC=/usr/local/gcc-6.2.0/bin/gcc ./configure \
-	--disable-bluez         --disable-brlapi \
-	--disable-bzip2         --disable-curl \
-	--disable-curses        --disable-debug-tcg \
-	--disable-fdt           --disable-glusterfs \
-	--disable-gtk           --disable-libiscsi \
-	--disable-libnfs        --disable-libssh2 \
-	--disable-libusb        --disable-linux-aio \
-	--disable-lzo           --disable-opengl \
-	--disable-rbd           --disable-qom-cast-debug \
-	--disable-rdma          --disable-sdl \
-	--disable-seccomp       --disable-slirp \
-	--disable-snappy        --disable-spice \
-	--disable-strip         --disable-tcg-interpreter \
-	--disable-tcmalloc      --disable-tools \
-	--disable-tpm           --disable-usb-redir \
-	--disable-uuid          --disable-vnc \
-	--disable-vte           --disable-vnc-{jpeg,png,sasl} \
-	--disable-xen           --enable-attr \
-	--enable-cap-ng         --enable-kvm \
-	--enable-virtfs         --target-list=x86_64-softmmu \
-	--extra-cflags="-fno-semantic-interposition -O3 -falign-functions=32" \
-	--datadir=/usr/local/share/qemu-lite \
-	--libdir=/usr/local/lib64/qemu-lite \
-	--libexecdir=/usr/local/libexec/qemu-lite
-$ make -j5 && sudo make install
-$ popd
 ```
+Note:
 
-## Install container kernel and rootfs
+If you are not using sudo, please ensure that your current user can run sudo without specifying a password:
```

Comment:
I'd make this even more explicit by adding a "Required Setup" section before "Installation steps" since, as some of us now know, it's very annoying to have this script fail after 40 minutes or so due to `sudo` not being setup correctly.

Further, you don't seem to have added the rest of the instructions from my comment about setting up the required directory structure - if you don't do this, the build will fail so please add all that to the "Required Setup" section too.